spec/focusvisibilitytracker.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { JSDOM } from 'jsdom';
+import { FocusVisibilityTracker } from '../src/ts/focusvisibilitytracker';
+
+describe('FocusVisibilityTracker', () => {
+  const bitmovinUIPrefix = 'bmpui';
+  const uiButtonId = `${bitmovinUIPrefix}-some-button`;
+  const nonUiButtonId = 'some-button';
+  const focusVisibileClassName = '{{PREFIX}}-focus-visible';
+  let uiButton: HTMLElement;
+  let nonUiButton: HTMLElement;
+  let tracker: FocusVisibilityTracker;
+
+  beforeEach(() => {
+    const { window } = new JSDOM();
+    global.document = window.document;
+    uiButton = createAndAddButton(document, uiButtonId);
+    nonUiButton = createAndAddButton(document, nonUiButtonId);
+    tracker = new FocusVisibilityTracker(bitmovinUIPrefix);
+  });
+
+  it('should add the focus-visible class on a UI button that gets focus w/o initial interaction', () => {
```

Comment:
I know we have differences in using the `should` word in test names 😄 (Up to you if you change that - No blocker still would prefer not using it) **all occurrences**
```suggestion
  it('adds the focus-visible class on a UI button that gets focus w/o initial interaction', () => {
```

Reply:
done in https://github.com/bitmovin/bitmovin-player-ui/pull/363/commits/bd031ac5d07276393ed1fc86a0f980df749206e6